demos/utils/cloud.py

```diff
@@ -68,6 +72,8 @@ def startCluster(self, testOutputFolder, logLevel="info",dataplane="envoy"):
         self.useCluster()
         super().set_kube_config()
         self.create_peer_cert(self.name,testOutputFolder)
+        if self.platform =="gcp" and self.machineType == "arm64": # Allow to schedule pods in arm64 on gcp
+            runcmd("kubectl taint nodes ` kubectl get nodes -o wide | awk '{if (NR>1) print $1}'` kubernetes.io/arch=arm64:NoSchedule-")
```

Comment:
I understand it is needed but I don't understand why GCP creates nodes with `NoSchedule` in the first place. Seems odd...

Reply:
This problem, I saw happen to other customers.